frontend/components/global/TimelineDropdown.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from "react";
+import {Dropdown, DropdownButton} from "react-bootstrap";
+
+const SORT_TYPES = {
+    CHRONOLOGICALLY: "Chronologically",
```

Comment:
Chronologically and most recent: is "most recent" the reverse of chronologically? Unclear.

Reply:
Thanks for feedback! We changed it to be "Chronological" and "Reverse Chronological"